CHANGELOG.md

```diff
@@ -31,6 +31,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Moved Cloud-J and Fast-JX input directories to Cloud-J and new Fast-JX menus respectively in `geoschem_config.yml`
 - Updated photolysis and aerosol optics input directories to use new mineral dust values in `FJX_scat-aer.dat` and `dust.dat` based on spheroidal shapes
 - Set `State_Diag%Archive_SatDiagn` to true if `State_Diag%Archive_SatDiagnPMID` is true
+- Disabled the `KppTime` diagnostic output the in `fullchem_alldiags` integration tests; this will vary from run to run causing difference tests to fail
```

Comment:
"the" and "in" are in the wrong order.

Reply:
Now fixed in https://github.com/geoschem/geos-chem/pull/2786/commits/0dcd0eaf0fb4a3d58eee5e0fc55c23167e651025